config/tools.json

```diff
@@ -7,15 +7,29 @@
         "description": "Explore APIs and companies with public AsyncAPI specifications.",
```

Comment:
I see only a single card updated - the one that was shown as an example in the screenshot in the linked issue. What about other 100+ cards?